package/upgrade/upgrade_manifests.sh

```diff
@@ -874,6 +897,31 @@ convert_monitoring_to_addon() {
         break
       fi
     done
+
+    # remove pv CliamRef when it is status 'Released', and after that, it becomes 'Available'
+    if [ $removeClaimRef = true ]; then
+      local phase=$(kubectl get pv $pvname -o jsonpath="{.status.phase}")
+      if [ $phase = "Released" ]; then
+        echo "patch ClaimRef of grafana pv, current status is Released"
+        echo "current ClaimRef is"
+        kubectl get pv $pvname -o jsonpath={".spec.claimRef"} && echo ""
+        # remove uid is enough
+        kubectl patch pv $pvname --type json -p '[{"op": "remove", "path": "/spec/claimRef/uid"}]'
+        sleep 2
+        echo "after patch, ClaimRef is"
+        kubectl get pv $pvname -o jsonpath={".spec.claimRef"} && echo ""
+        echo "the new status is"
+        kubectl get pv $pvname -o jsonpath="{.status.phase}" && echo ""
+      elif [ $phase = "Available" ]; then
+        echo "grafana pv status has already been Available, do not patch ClaimRef"
+        echo "current ClaimRef is"
+        kubectl get pv $pvname -o jsonpath={".spec.claimRef"} && echo ""
```

Comment:
If the PV's phase is Available, will it have a claimRef?

Reply:
When the `ClaimRef` is nil, or `ClaimRef.UID` is `""`, the PV will be `Available`.

It is possible that, `ClaimRef` is not nil, but `ClaimRef.UID` is `""`. The print will help us to know more details.